backend/src/utils/cors.ts

```diff
@@ -2,6 +2,6 @@ export function cors(): object {
 	return {
 		'Access-Control-Allow-Origin': '*',
 		'Access-Control-Allow-Headers': 'content-type, authorization, idempotency-key',
-		'Access-Control-Allow-Methods': 'GET, PUT, POST',
+		'Access-Control-Allow-Methods': 'GET, PUT, POST, DELETE',
```

Comment:
Although this will enable the DELETE request to get past CORS, I don't believe that deletes are yet implemented in Wildebeest.

Reply:
Ah, yes, it just failed silently...